aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/AqualityUncheckedException.java

```diff
@@ -0,0 +1,18 @@
+package aquality.tracking.integrations.core;
+
+import java.io.IOException;
+
+public class AqualityUncheckedException extends RuntimeException {
```

Comment:
do we really need it? maybe it can be simpler to use IllegalStage/IllegalArg/... exceptions?

Reply:
Having this exception allows to better identify problem when you use this plugin in tests. You see that `AqualityUncheckedException` occurred and know that the problem relates to this integrations.